### PR TITLE
Fetch comment and crdb_sql_type in single query for table

### PR DESCRIPTION
**Pull request by carlallen**
Solves #234

### CODE REVIEW

**Comment by keithdoggett** [comment]
Looks like comments aren't being picked up correctly anymore based on the output from the CI. Also, it seems like there may be some differences in the output that are causing intervals and geographic types to be parsed improperly. You might have to `downcase` the type to fix that. Not sure what's going on with the comments right now.

**Review by keithdoggett — APPROVED** [review]
LGTM thanks for the work on this

File: lib/active_record/connection_adapters/cockroachdb_adapter.rb [hunk 1]
```diff
@@ -435,32 +449,33 @@ def column_definitions(table_name)
             /interval/,
             /numeric/
           ]
+
           re = Regexp.union(target_types)
```
Comment (rafiss) [inline lib/active_record/connection_adapters/cockroachdb_adapter.rb:453]: is `target_types` and this `regexp` unused now?

**Review by rafiss — CHANGES_REQUESTED** [review]

File: lib/active_record/connection_adapters/cockroachdb_adapter.rb (hunk 1 above)
Comment (carlallen) [reply lib/active_record/connection_adapters/cockroachdb_adapter.rb:453]: Yes it's in the condition on https://github.com/cockroachdb/activerecord-cockroachdb-adapter/pull/235/files#diff-aa6cbad03effcf32e5ea1d8867d8727eb6a605525828574bd3b2ef8c249f8f54R456

Comment (rafiss) [reply lib/active_record/connection_adapters/cockroachdb_adapter.rb:453]: missed that, thanks!

**Review by rafiss — APPROVED** [review]
lgtm!

**Pull request merged**